lib/draft/ranges.ex

```diff
@@ -5,30 +5,28 @@ defmodule Draft.Ranges do
       Provides functions for adding inline style ranges and entity ranges
       """
 
-      def apply_ranges(text, inline_style_ranges, entity_ranges, entity_map, context) do
-        inline_style_ranges ++ entity_ranges
-        |> consolidate_ranges()
-        |> Enum.reduce(text, fn {start, finish}, acc ->
-          {style_opening_tag, style_closing_tag} =
-            case get_styles_for_range(start, finish, inline_style_ranges, context) do
-              "" -> {"", ""}
-              styles -> {"<span style=\"#{styles}\">", "</span>"}
-            end
-          entity_opening_tags = get_entity_opening_tags_for_start(start, entity_ranges, entity_map, context)
-          entity_closing_tags = get_entity_closing_tags_for_finish(finish, entity_ranges, entity_map, context)
-          opening_tags = "#{entity_opening_tags}#{style_opening_tag}"
-          closing_tags = "#{style_closing_tag}#{entity_closing_tags}"
+      def apply_ranges(block, entity_map, context) do
+        block["inlineStyleRanges"]
+        |> divvy_style_ranges(block["entityRanges"])
+        |> group_style_ranges()
+        |> Kernel.++(block["entityRanges"])
+        |> sort_by_offset_and_length_then_styles_first()
```

Comment:
Obj-C naming is leaking

Reply:
I'm unapologetic 😂 

Unless you want me to be apologetic.

Then I'll be apologetically apologetic. 😏 🙇